plugins/tiddlywiki/tiddlyweb/sse-server.js

```diff
@@ -0,0 +1,62 @@
+/*\
+title: $:/plugins/tiddlywiki/tiddlyweb/sse-server.js
+type: application/javascript
+module-type: route
+
+GET /events/plugins/tiddlywiki/tiddlyweb
+
+\*/
+(function(){
+
+/*jslint node: true, browser: true */
+/*global $tw: false */
+"use strict";
+
+/** @type {Record<any, {
+ *   request: import("http").IncomingMessage,
+ *   state: {wiki,pathPrefix},
+ *   emit: (event: string, data: string) => void,
+ *   end: () => void
+ * }[]>} */
+var wikis = {};
+/**
+ * Setups up the array for this wiki and adds the change listener
+ * 
+ * @param {$tw.Wiki} wiki The wiki object to listen to changes on
+ */
+function setupWiki(wiki) {
+  // add a new array for this wiki (object references work as keys)
+  wikis[wiki] = [];
```

Comment:
I think this would work ok for a single wiki, but IIRC JavaScript will stringify the object reference, so you'll end up with something like `wikis['[object Object]'] = []`, right?

Reply:
You're right. Many thanks. Fixed.